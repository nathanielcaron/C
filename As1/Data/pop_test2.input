u 10
u 20
o
o
x