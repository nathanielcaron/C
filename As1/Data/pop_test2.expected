10
20
20
10
Successfully executed 4 instructions
0 elements
